examples/react-todos/client/components/accounts/UserSidebarSection.jsx

```diff
@@ -6,9 +6,11 @@ UserSidebarSection = React.createClass({
       menuOpen: false
     };
   },
+
```

Comment:
I have read a lot of parts of it, but AFAIK code complete doesn't have much to say about spacing inside React component definitions. Although I guess it just generalizes to "classes".

Great book, BTW